pageserver/src/tenant/timeline/span.rs

```diff
@@ -1,26 +1,25 @@
+use utils::tracing_span_assert::{check_fields_present, Extractor, MultiNameExtractor};
+
 #[cfg(not(debug_assertions))]
 #[inline]
```

Comment:
Let's remove this as well, no need to profilirate these.
```suggestion
```

Reply:
Done